answerking_app/utils/mixins/ApiExceptions.py

```diff
@@ -3,6 +3,10 @@
 from rest_framework.exceptions import APIException
 
 
+class BadInputParameter(Exception):
+    pass
```

Comment:
should this be a pass?

Reply:
why was this object created? HTTPErrorResponse can be raised